pkg/cmd/server/start/master_args.go

```diff
@@ -182,6 +185,19 @@ func (args MasterArgs) BuildSerializeableMasterConfig() (*configapi.MasterConfig
 
 	dnsServingInfo := servingInfoForAddr(&dnsBindAddr)
 
+	overrideConfig := configapi.ClusterResourceOverrideConfig{}
```

Comment:
I don't like this section.  The intent of this method is to build a serializeable config from arguments.  It should be entirely predictable and I don't think that it should allow an admission plugin config file to override the output of the actual fix section of the config at all.

@liggitt I've been out in rebase-land for a while.  Is this something you inspected already?